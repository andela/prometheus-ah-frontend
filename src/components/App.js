import React, { Component } from 'react';

/**
 * @class App
 */
class App extends Component {
  render() {
    return (
      <div>
        <h1>Welcome to Authors haven</h1>
      </div>
    )
  }
}

export default App;
