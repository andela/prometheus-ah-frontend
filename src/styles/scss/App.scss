@import "./base/settings";
@import "./base/base";
@import "./components/form";
@import "./components/modal";
@import "./components/article";
@import "./components/navigation";
@import "./components/carousel";
@import "./components/featuredPost";
@import "./components/recentPost";
@import "./components/featuredAuthor";
