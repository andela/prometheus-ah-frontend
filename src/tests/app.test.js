import React from 'react';
import Enzyme, { shallow } from 'enzyme';
import Adapter from 'enzyme-adapter-react-16';
import App from '../components/App';

Enzyme.configure({ adapter: new Adapter() });

describe('App component', () => {
  test('renders', () => {
    const wrapper = shallow(<App />);
    expect(wrapper.exists()).toBe(true);
  });

  test('renders a snapshot of the app component', () => {
    const wrapper = shallow(<App />);
    expect(wrapper).toMatchSnapshot();
  });
});
