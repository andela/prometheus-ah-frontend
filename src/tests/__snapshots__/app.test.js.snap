// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`App component renders a snapshot of the app component 1`] = `
<div>
  <h1>
    Welcome to Authors haven
  </h1>
</div>
`;
